src/App.css

```diff
@@ -1,5 +1,249 @@
+}
+
+.chapter_status {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.content_container {
+  display: flex;
+  justify-content: space-between;
+}
+
+.span_container {
+  display: flex;
+  flex-direction: column;
+}
+
+.card {
+  width: 100%;
+  margin-top: 5%;
+  margin-bottom: 5%;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+  transition: 0.4s;
+  padding: 3%;
+}
+
+.category_p {
+  padding-left: 2%;
+  font-weight: bold;
+  color: #121212;
+}
+
+.title_p {
+  padding-left: 2%;
+  font-weight: bold;
+  font-size: 200%;
+}
+
+.author {
+  padding-left: 2%;
+  color: #4386bf;
+  margin-bottom: 20%;
+}
+
+.card_containment {
+  display: flex;
+}
+
+/* form */
+.add_book_p {
+  font-weight: bold;
+  font-size: 200%;
+  color: #888;
+}
+
+/* button */
+.button_ul {
+  padding-left: 0%;
+}
+
+small {
+  color: #888;
+}
+
+.button_container {
+  display: flex;
+}
+
+button {
+  font-weight: bold;
+  border: none;
+  border-radius: 4px;
+}
+
+.update_button {
+  border-radius: 3px;
+  background-color: #0290ff;
+  color: #fff;
+  font-weight: normal;
+}
+
+.submit_button {
+  /* width: 11.5rem;
+  height: 2.813rem;
+  margin: 1.188rem 0 0 2.125rem;
+  padding: 0.801rem 1.188rem 0.886rem 1.375rem; */
+  border-radius: 3px;
+  background-color: #0290ff;
+  width: 30%;
+}
+
+.remove_button {
+  background: #fff;
+  color: #4386bf;
+  font-weight: normal;
+}
+
+/* inputs */
+.title_input,
+.category_input {
+  /* width: 41.5rem;
+  height: 2.813rem;
+  margin: 1.188rem 2.125rem 0 0;
+  padding: 0.813rem 35.688rem 0.813rem 1.063rem; */
+  width: 100%;
+  border-radius: 4px;
+  border: solid 1px rgba(0, 0, 0, 0.05);
+  padding: 1%;
+  margin-right: 2%;
+}
+
+.category_input {
+  width: 50%;
+}
+
+form {
+  margin-bottom: 5%;
+  display: flex;
+}
+
+/* navbar */
+.navContainer {
+  background-color: #fff;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.heading {
+  font-size: 250%;
+  font-weight: bold;
+  color: #0290ff;
+}
+
+.navBarUl {
+  list-style-type: none;
+  padding-left: 0%;
+  display: flex;
+  align-items: baseline;
+}
+
+.navBarUl > li {
+  padding: 2%;
+  padding-left: 10%;
+}
+
+.profile_icon {
+  margin-right: 2%;
+}
+
+.navLink {
+  text-decoration: none;
+  font-size: 115%;
+  color: #000;
+}
+
+/* progress circle */
+.circle_status {
+  margin-left: 20%;
+}
+
+.circle-wrap {
+  margin: 50px auto;
+  width: 150px;
+  height: 150px;
+  background: #e6e2e7;
+  border-radius: 50%;
+}
+
+.circle-wrap .circle .mask,
+.circle-wrap .circle .fill {
+  width: 150px;
+  height: 150px;
+  position: absolute;
+  border-radius: 50%;
+}
+
+.circle-wrap .circle .mask {
+  clip: rect(0, 150px, 150px, 75px);
+}
+
+.circle-wrap .circle .mask.full,
+.circle-wrap .circle .fill {
+  animation: fill ease-in-out 3s;
+  transform: rotate(126deg);
+}
+
+.circle-wrap .circle .mask .fill {
+  clip: rect(0, 75px, 150px, 0);
+  background-color: #9e00b1;
+}
```

Comment:
- [ ] Well done so far, however, the background color you used for the circular progress bar is quite different from the Zeplin's. Let's make use of the blue color `#0290ff;` that you have been using instead of this color `#9e00b1;`. This is to make sure that you meet the project requirement as your design has to match that of Zeplin. 
- [ ] Let's have a `div` that will wrap the circular progress bar and another `div` that will wrap the progress percentage `64%`  and the completed paragraph `completed` and display the main `div` as `flex` so that you will have the circular progress bar and the percentage completed side by side.

Something like this:
``` 
<div>
    <div> circular progress bar</div>
    <div>
        <p> 68% <p>
        <p>completed </p>
     </div>
</div>
```


Your Implementation | Figma Design
:---:|:---:
![](https://i.imgur.com/L8llHwe.png) | ![](https://i.imgur.com/oLxBXcj.png)

Reply:
Thank you for your observation. I just feel like this should not be something to block my progress. I feel like it should be an optional suggestion. Not that I am a code reviewer or anything but it gets frustrating when a project is blocked for such a minor fix. I am left with one code review and I am at risk of repeating the week. On top of that, I have not done Friday's activities because of working on this. I am not trying to be rude in any way, I just felt like letting this out. In any case, you are an observant and awesome reviewer😎 and I have implemented the changes you require. 💻